src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -28,18 +28,22 @@
 
 struct WayPoint
 {
-    WayPoint(uint32 _id, float _x, float _y, float _z)
+    WayPoint(uint32 _id, float _x, float _y, float _z, float _o, uint32 _delay)
```

Comment:
Is this struct used somewhere else? If yes, it should be better to set the last parameters to 0 as default.

Reply:
this struct was throwinging up the compile test errors, apparrently it is related to the adjustments in adding the orentation and the delay which is definately a bit different from what hte original pr on tc showed. at this point, it is required for compile testing from what i can tell. I do not use linux or macos, so the error present on those i would not be able to verify first hand.